extensions/spring-aot.md

```diff
@@ -0,0 +1,43 @@
+# Spring Ahead of Time
+
+[Spring AOT processing](https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html#native-image.introducing-graalvm-native-images.understanding-aot-processing)
+is part of the process to create a native binary from a Spring (Boot) application. This extension will help in adding a
+lot of hints which are needed. Please note this extension can only be used with Spring Boot 3, as
+such it requires at least Java 17.
+
+Besides the extension it might be needed to make more changes to successfully compile and run an application as native
+image. For example, when a message isn't use in a handler. This is quite common when the application is split, and the
```

Comment:
```suggestion
Besides the extension, it might be necessary to make more changes to successfully compile and run an application as a native
image. For example, when a message isn't used in a handler. This is quite common when the application is split, and the
```